azure-pipelines.yml

```diff
@@ -73,9 +79,12 @@ jobs:
     inputs:
       verbose: false
   - script: 'scripts/install-dotnet.sh'
+    displayName: 'install .Net'
   - script: 'node scripts/momentOfTruth.js && node scripts/momentOfTruthPostProcessing.js'
+    displayName: 'LintDiff'
 
 - job: "SDK"
+  timeoutInMinutes: 120
```

Comment:
oh wow, just curious is there a sample of something that comes even close to 2 hours ?

Reply:
yes, Network https://github.com/Azure/azure-rest-api-specs/pull/5266